lib/pengine/pe_output.c

```diff
@@ -382,7 +382,7 @@ pe__cluster_summary(pcmk__output_t *out, va_list args) {
                                    crm_element_value(dc_version, XML_NVPAIR_ATTR_VALUE)
                                    : NULL;
         const char *quorum = crm_element_value(data_set->input, XML_ATTR_HAVE_QUORUM);
-        char *dc_name = data_set->dc_node ? pe__node_display_name(data_set->dc_node, print_clone_detail) : NULL;
+        char *dc_name = data_set->dc_node ? pe__node_display_name(data_set->dc_node, pcmk_is_set(print_opts, pe_print_clone_details)) : NULL;
```

Comment:
This might be a good opportunity to fix an issue that has crept in over time. Initially the crm_mon --show-detail option only set pe_print_clone_details, and thus its only effect was showing individual clone instances.

However over time we've expanded --show-detail to also show node IDs (the code above here) and implicit resources (via setting pe_print_implicit).

This would be a good time to clear that up. A new display option (something like pe_print_node_ids) should control whether node IDs are displayed. crm_mon should name its variable generically (something like print_detail), and then then just set the appropriate display options.

Reply:
I think I've taken care of this.